cypress/integration/examples/cypress_api.spec.js

```diff
@@ -158,7 +164,7 @@ context('Cypress.env()', () => {
     // set multiple environment variables
     Cypress.env({
       host: 'veronica.dev.local',
```

Comment:
move this to a local variable since it's reused

Reply:
Removed since this is sample test only